frontend/src/components/Board/Board.tsx

```diff
@@ -38,23 +38,9 @@ const members = [
 const InnerBoard: React.FC = () => {
   // Example Board
   // http://localhost:5173/Board/02b5bfae-f543-42b6-aab0-51b39253a2e4
-  const { board_id } = useParams();
+  const { board_id } = useParams() as { board_id: string };
 
-  const boardResult = useBoard(board_id);
-  let board: Board = {
-    created_at: null,
-    description: null,
-    id: '',
-    name: null,
-    saved_date: null,
-    team_id: null,
-    user_id: null,
-    version: -1,
-  };
-  if (boardResult?.data?.length > 0) {
-    board = boardResult?.data?.slice(-1)[0];
-    // console.log({ board })
-  }
+  const { data, isLoading, error, refetch, isRefetching } = useBoard(board_id);
```

Comment:
are these just the standard return type properties from supabase? I don't get the point of isrefetching lol

Reply:
yep. `isRefetching` is used because when the `refetch` question is called, `isLoading` will be false and is not updated.